melodies_monet/driver.py

```diff
@@ -427,6 +480,10 @@ def mask_and_scale(self):
                             self.obj[v].data += scale
                         elif d['unit_scale_method'] == '-':
                             self.obj[v].data += -1 * scale
+                    if self.obj[v].units == 'ppv':
+                        print('changing units for {}'.format(v))
+                        self.obj[v].values *= 1e9
+                        self.obj[v].attrs['units'] = 'ppbv'        
```

Comment:
If a user converts with unit_scale_method - then the units should also change and no longer be 'ppv', so this if statement should not be implemented.

Reply:
As far as I can tell there is no relabelling of `self.obj[v].units` associated with using unit_scale_method. I agree that it would be good to implement something like that eventually but currently I don't see where `self.obj[v].units` would change when using unit_scale_method, so this if statement would run even if the unit_scale_method section above it has also run.

Reply:
We would like to defer unit conversion issues to a future discussion and code branch.  We have created an issue for this https://github.com/NOAA-CSL/MELODIES-MONET/issues/192